### PR TITLE
Less : bump

### PR DESCRIPTION
**Pull request by extrowerk**

### CODE REVIEW

File: sys-apps/less/less-530.recipe [hunk 1]
```diff
@@ -6,15 +6,15 @@ large input files it starts up faster than text editors like 'vi'.
...
 LICENSE="GNU GPL v3"
 REVISION="1"
 SOURCE_URI="http://www.greenwoodsoftware.com/less/less-$portVersion.tar.gz"
-CHECKSUM_SHA256="dba42cd4c38174b7bd0d426d8a39df2be6bcc1ec18946a4642713290f6bf9a0b"
+CHECKSUM_SHA256="503f91ab0af4846f34f0444ab71c4b286123f0044a4964f1ae781486c617f2e2"
 PATCHES="less-$portVersion.patchset"
 ADDITIONAL_FILES="sysless.in"
 
-ARCHITECTURES="?x86_gcc2 ?x86 ?x86_64"
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
```
Comment (fbrosson) [inline sys-apps/less/less-530.recipe]: Oops, we need to keep at least one recipe with x86_gcc2 marked as tested.

**Comment by extrowerk** [comment]
Can somebody test the current on gcc2?

-- Z.

2018.05.01. dátummal, 17:47 időpontban fbrosson <notifications@github.com> írta:

File: sys-apps/less/less-530.recipe (hunk 1 above)
Comment (fbrosson) [reply sys-apps/less/less-530.recipe]:
```diff
-ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+ARCHITECTURES="x86_gcc2 ?x86 x86_64"
```
(less-530 builds and seems to be working fine on x86_gcc2.)

**Comment by fbrosson** [comment]
I could not resist syncing the patch... and marking x86 as tested :-)
(I'm playing with the bootstrap builds, and having less-530 marked as tested on x86 will be cool ;)

**Pull request merged**